server.js

```diff
@@ -1,60 +1,60 @@
-// TASK 1: Setting up server
+// // TASK 1: Setting up server
 
 
-const http = require('http');
-
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
```

Comment:
The PORT variable should be changed to a different value since port 80 is a well-known port and may be reserved by the system.